### DISCUSSION TITLE
Is strict mode supported or not? (Firefox)

### DISCUSSION THREAD

**Discussion by Xpower7** [Cookie-AutoDelete/Cookie-AutoDelete#1568] [Q&A / Support]
"As of Firefox 86, strict mode is not supported at this time due to missing APIs to handle the Total Cookie Protection. Either use standard mode (this would have been the 'strict' mode pre-86), or use the custom configuration and set cookie to 'cross site tracking cookies' option (not the cross-site cookies)."

That message on the FF page is very confusing, specially when english is not my first language. Is that message relevant today? We are way ahead of version 86.

**Comment by kennethtran93** [comment]
Strict mode continues to not be supported at this time.  Any updates will go into #807.

In recent versions they also applied the Total Cookie Protection as standard as well, so the webextension is partially working at this time.
